package.json

```diff
@@ -9,20 +9,28 @@
   "main": "gulpfile.js",
   "dependencies": {},
   "scripts": {
+    "dev": "gulp watch",
```

Comment:
@bahiirwa Once we will be done with the PR review, and once this PR will be ready for merge, please create a `CONTRIBUTING.md` file and document how to use the new build system, and what to do for transifex etc... I am sure your documentation expertise will make it easy for new contributors ;)